_about/notes/index.md

```diff
@@ -16,17 +16,18 @@ toc: false
 
 {% include section-index.html docs=site.about %}
 
+The [latest Istio monthly release](https://github.com/istio/istio/releases) is {{site.data.istio.version}} which
+you can download with:
 
-- The [latest](https://github.com/istio/istio/releases) Istio monthly release is {{site.data.istio.version}}. It is downloaded when the following is used(*):
-  ```
-  curl -L https://git.io/getLatestIstio | sh -
-  ```
+```
+curl -L https://git.io/getLatestIstio | sh -
+```
 
-- The most recent 'stable' release is [0.2.12](https://github.com/istio/istio/releases/tag/0.2.12), the matching docs are [archive.istio.io/v0.2/docs/](https://archive.istio.io/v0.2/docs/)
-  ```
-  curl -L https://git.io/getIstio | sh -
-  ```
+The most recent stable release is [0.2.12](https://github.com/istio/istio/releases/tag/0.2.12) which you
+can download with:
 
-We typically wait to 'bake' the latest release for several weeks and ensure it is more stable than the previous one before promoting it to stable.
+```
+curl -L https://git.io/getIstio | sh -
+```
 
-> (*) Note: security conscious users should examine the output of the curl command before piping it to a shell.
+[Archived documentation for for 0.2.12 release](https://archive.istio.io/v0.2/).
```

Comment:
typo (for for) fixed in https://github.com/istio/istio.github.io/pull/1115/files